src/fal/planner/plan.py

```diff
@@ -28,8 +29,17 @@ def from_execution_plan(
     ) -> FilteredGraph:
         graph = origin_graph.copy_graph()
         for node in origin_graph.graph.nodes:
-            if node not in execution_plan.dbt_models:
+            if (
+                node
+                not in execution_plan.dbt_models
+                + execution_plan.before_scripts
+                + execution_plan.after_scripts
```

Comment:
can you parenthesize this expression? I am guessing it is `not in (list+list+list)`